objectPerformance/src/plotter.py

```diff
@@ -247,29 +246,126 @@ def _compute_value_of_tanh_at_threshold(self,
                                             efficiency,
                                             bins,
                                             threshold):
-        xvals = [x - (bins[1] - bins[0]) / 2 for x in bins[1:]]
-        popt, pcov = curve_fit(utils.tanh, xvals, efficiency, p0=[1,0])
+        xvals = bins
+        xvals = np.array(xvals); efficiency = np.array(efficiency)
```

Comment:
pep8 discourages having multiple statements in one line, which I would agree with.
```
xvals = bins
xvals = np.array(xvals)
```
is the same as `xvals = np.array(bins)`